rust-module/mfrc522_inner/spi.rs

```diff
@@ -125,6 +134,14 @@ impl Mfrc522Spi {
         Ok(())
     }
 
+    pub fn fifo_flush(dev: &mut SpiDevice) -> KernelResult {
+        let flush_byte: u8 = 1u8 << 7;
```

Comment:
constify this like so: https://github.com/ks0n/mfrc522-linux/blob/085c8fe48cea6cf87103bd108c80d562bc25f974/rust-module/mfrc522_inner/command.rs#L1

Reply:
> Could you also add `spi_dev` to mem_write ? That will make a more uniform interface for all commands

I think we can do this later, I opened an issue already #57. This way we can merge this and keep going as this is mostly "maintenance"